packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -1,12 +1,11 @@
 <script lang="typescript">
     import { localize } from '@core/i18n'
-    import { WalletAccount } from '@lib/typings/walletAccount'
     import { ADDRESS_LENGTH, validateBech32Address } from '@lib/utils'
     import { RecipientAccountSelector, TextInput, InputContainer, Modal } from 'shared/components'
     import { activeProfile } from '@core/profile'
     import { getNetwork, NetworkType, nodeInfo } from '@core/network'
-
-    export let recipient: string | WalletAccount
+    import { IAccountState } from '@core/account'
+    export let recipient: string | IAccountState
```

Comment:
Maybe we can put a new line here to separate props from imports

Reply:
Will do

Reply:
Done